src/please.go

```diff
@@ -215,6 +215,11 @@ var opts struct {
 				Options ConfigOverrides `positional-arg-name:"config" required:"true" description:"Attributes to set"`
 			} `positional-args:"true" required:"true"`
 		} `command:"config" description:"Initialises specific attributes of config files"`
+		Pleasings struct {
```

Comment:
would it be better to prompt interactively during `plz init` rather than having a separate subcommand?

Reply:
Ah yeah that's a good way to do this. I think this is still useful for migrating an existing repository though (once they see the warning above).